src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -93,12 +93,16 @@ namespace
         ParamIdProjectPath                  = 1,
         ParamIdScaleMultiplier              = 2,
 
-        ParamIdPixelSamples                 = 3,
+        ParamIdUniformPixelSamples          = 3,
         ParamIdTileSize                     = 4,
         ParamIdPasses                       = 5,
         ParamIdFilterSize                   = 6,
         ParamIdFilterType                   = 7,
-
+        ParamIdSamplerType                  = 40,
```

Comment:
Maybe `ImageSampler` instead of `Sampler`?

Reply:
True, ImageSampler is more verbose.